startup.sh

```diff
@@ -1,14 +0,0 @@
-export CPATH=/root/tpm2/utils
-export LD_LIBRARY_PATH=/root/tpm2/utils:/root/tpm2/utils12“
-export TPM_DATA_DIR=/root/tpm2
-export TPM_SERVER_TYPE=mssim
-export ACS_PORT=2323
-
-mkdir /tmp/mytpm1
-swtpm_setup --tpmstate /tmp/mytpm1 --create-ek-cert --create-platform-cert  --tpm2 --display
-swtpm socket --tpmstate dir=/tmp/mytpm1 --ctrl type=unixio,path=/tmp/mytpm1/swtpm-sock --log level=20 --tpm2 --server type=tcp,port=2321 &
-sleep 3 
-swtpm_ioctl -i --unix /tmp/mytpm1/swtpm-sock
```

Comment:
Why delete this file? If you do Not need this file, just do not add in previous commit https://github.com/inclavare-containers/confidential-vtpm/pull/4/commits/49566b33d6c59cc6a6c71baca0991472a6320da7

Reply:
I have already deleted this file in the previous commit